spark-connector/src/main/java/com/datastrato/gravitino/spark/GravitinoSparkConfig.java

```diff
@@ -0,0 +1,15 @@
+/*
+ *  Copyright 2024 Datastrato Pvt Ltd.
+ *  This software is licensed under the Apache License version 2.
+ */
+
+package com.datastrato.gravitino.spark;
+
+public class GravitinoSparkConfig {
+  private static final String GRAVITINO_PREFIX = "spark.sql.gravitino.";
+  public static final String GRAVITINO_URI = GRAVITINO_PREFIX + "uri";
+  public static final String GRAVITINO_METALAKE = GRAVITINO_PREFIX + "metalake";
+
+  public static final String GRAVITINO_HIVE_METASTORE_URI = "metastore.uris";
```

Comment:
Why do we need this? Could we reuse the constants of Hive or Spark?

Reply:
`metastore.uris` is a gravitino hive metastore uri,  it's used by Gravitino Hive catalog, Trino-connector and spark-connector,  the names are not well managed for now, I prefer to manage common names it in separate PR.

`hive.metastore.uris` is the config defined in HiveConfig, introduce hive depends to spark-connector just using a variable seems a little heavy.